Gruntfile.js

```diff
@@ -2,39 +2,41 @@ const webpackConfig = require('./webpack.config')
 
 module.exports = function (grunt) {
   grunt.loadNpmTasks('grunt-contrib-clean')
-  grunt.loadNpmTasks('grunt-contrib-uglify')
   grunt.loadNpmTasks('grunt-webpack')
 
   grunt.initConfig({
     clean: ['dist'],
-    uglify: {
-      my_target: {
-        files: {
-          'dist/leanplum.min.js': 'dist/leanplum.js',
-          'dist/sw/sw.min.js': 'dist/sw/sw.js'
-        }
-      }
-    },
     watch: {
       js: {
         files: [
           'lib/leanplum.js'
         ],
-        tasks: ['lint', 'compile']
+        tasks: ['build']
       }
     },
```

Comment:
It looks like the watcher waits for changes in `lib/leanplum.js` (which does not exist) to run the `build` command. I don't think that we need this as a workflow since we're mostly running tests against `src/`. Consider deleting.

Reply:
Thanks, removed as suggested!